test/Mesh/interpolation.jl

```diff
@@ -266,9 +266,9 @@ function run_cubed_sphere_interpolation_test()
         x1g = Array{FT}(undef,nrad,nlat,nlong); x2g = similar(x1g); x3g = similar(x1g)
 
         for k in 1:nlong, j in 1:nlat, i in 1:nrad
-            x1g[i,j,k] = rad[i] * sin(lat[j]) * cos(long[k]) # inclination -> latitude; azimuthal -> longitude.
-            x2g[i,j,k] = rad[i] * sin(lat[j]) * sin(long[k]) # inclination -> latitude; azimuthal -> longitude.
-            x3g[i,j,k] = rad[i] * cos(lat[j])
+            x1g[i,j,k] =  rad[i] * cosd(lat[j]) * cosd(long[k]) # inclination -> latitude; azimuthal -> longitude.
+            x2g[i,j,k] =  rad[i] * cosd(lat[j]) * sind(long[k]) # inclination -> latitude; azimuthal -> longitude.
+            x3g[i,j,k] = -rad[i] * sind(lat[j])
```

Comment:
Why is this negated?

Reply:
Inclination angle originally measured from z axis. Definition filled to use -pi/2 at south pole & pi/2 at North pole.